src/translations/en.json

```diff
@@ -1044,7 +1044,7 @@
         },
         "automation": {
           "caption": "Automations",
-          "description": "Manage automations",
+          "description": "e.g. 'Turn off the lights when I leave home'",
```

Comment:
"Create and modify automations" IMHO would be better. Examples are not descriptions.

Reply:
The problem I'm trying to solve: as a new user, I have no idea what an automation *is*.

I'm in the Configure section, and the caption is Automations, so I pretty much expect that clicking on this will let me configure (i.e. create and modify) automations.

I think examples are a great way to convey what something actually is. home-assistant.io does a pretty good job of describing automations on the home page with an example:

> Automate: Set up advanced rules to control devices and bring your home alive. Do you want to … have the lights turn on when the sun sets and you are home?

So: that's my argument for why an example makes the best sense here.

Reply:
Done.